--- atividades/0104GEO001/0104GEO00104.js	2012-06-01 14:43:00.244690012 -0300
+++ atividades/0104GEO001/0104GEO00104.js.new	2012-06-01 15:12:51.560716929 -0300
@@ -445,7 +445,7 @@
 
 
 
-	PreloadImages('../imagens/logo.png','Paraiba.gif','Sergipe.gif','Alagoas.gif','Bahia.gif','rodape.png');
+	PreloadImages('../imagens/logo.png','paraiba.gif','sergipe.gif','alagoas.gif','bahia.gif','rodape.png');
 
 
 	SetUpItems(ShuffleQs,QsToShow);
